users.js

```diff
@@ -321,15 +321,20 @@ module.exports = function (options) {
 
   function cmd_reset_password (args, done) {
     var seneca = this;
-    seneca.act({role: 'auth', cmd: 'create_reset'}, args, function (err, response) {
-      if (err) return done(err);
-      return done(null, response);
-    });
+    args.data = {};
```

Comment:
Format for args passed by the call at https://github.com/CoderDojo/cp-users-service/pull/195/files#diff-10aafeb1e619811e8b26a126d6ca2666R327